src/app/Scenes/MyCollection/Screens/Insights/AverageSalePriceSelectArtistItem.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  AverageSalePriceSelectArtistItem_artist$data,
+  AverageSalePriceSelectArtistItem_artist$key,
+} from "__generated__/AverageSalePriceSelectArtistItem_artist.graphql"
+import OpaqueImageView from "app/Components/OpaqueImageView/OpaqueImageView"
+import { Flex, NoArtworkIcon, Text, Touchable, useColor } from "palette"
+import React from "react"
+import { useFragment } from "react-relay"
+import { graphql } from "relay-runtime"
+import { AverageSalePriceArtistType } from "./AverageSalePriceSelectArtist"
+
+interface ArtistSectionItemProps {
+  artist: AverageSalePriceArtistType
+  onPress: (artist: AverageSalePriceSelectArtistItem_artist$data) => void
+  isFirst?: boolean
+}
+
+export const ArtistSectionItem: React.FC<ArtistSectionItemProps> = ({
+  isFirst,
+  onPress,
+  ...restProps
+}) => {
+  const color = useColor()
+  const artist = useFragment<AverageSalePriceSelectArtistItem_artist$key>(
```

Comment:
No need to type `useFragment` 🥳 
```suggestion
  const artist = useFragment(
```